data/tlg0341/tlg002/tlg0341.tlg002.perseus-grc1.xml

```diff
@@ -1,1606 +1,1604 @@
+                <l n="981">κεῖται, κάρα μάστιγι γογγύλῃ τυπείς,</l>
+                <l n="982">ῥείθροισιν ὠκὺς ἔνθα μύρεται Σίνις,</l>
+                <l n="983">ἄρδων βαθεῖαν Χωνίας παγκληρίαν.</l>
+                <l n="984">πόλιν δʼ ὁμοίαν Ἰλίῳ δυσδαίμονες</l>
+                <l n="985">δείμαντες, ἀλγυνοῦσι Λαφρίαν κόρην </l>
+                <l n="986">σάλπιγγα, δῃώσαντες ἐν ναῷ θεᾶς</l>
+                <l n="987">τοὺς πρόσθʼ ἔδεθλον Ξουθίδας ᾠκηκότας.</l>
+                <l n="988">γλήναις δʼ ἄγαλμα ταῖς ἀναιμάτοις μύσει,</l>
+                <l n="989">στυγνὴν Ἀχαιῶν εἰς Ἰάονας βλάβην</l>
+                <l n="990">λεῦσσον φόνον τʼ ἔμφυλον ἀγραύλων λύκων, </l>
+                <l n="991">ὅταν θανὼν λῄταρχος ἱρείας σκύλαξ</l>
+                <l n="992">πρῶτος κελαινῷ βωμὸν αἱμάξῃ βρότῳ.</l>
+                <l n="993">ἄλλοι δὲ πρῶνας δυσβάτους Τυλησίους</l>
+                <l n="994">λίνου θʼ ἁλισμήκτοιο δειραίαν ἄκραν</l>
+                <l n="995">Ἀμαζόνος σύγκληρον ἄρσονται πέδον, </l>
+                <l n="996">δούλης γυναικὸς ζεῦγλαν ἐνδεδεγμένοι,</l>
+                <l n="997">ἣν χαλκομίτρου θῆσσαν ὀτρηρῆς κόρης</l>
+                <l n="998">πλανῆτιν ἄξει κῦμα πρὸς ξένην χθόνα.</l>
+                <l n="999">ἧς ἐκπνεούσης λοῖσθον ὀφθαλμὸς τυπεὶς</l>
+                <l n="1000">πιθηκομόρφῳ πότμον Αἰτωλῷ φθόρῳ </l>
+                <l n="1001">τεύξει τράφηκι φοινίῳ τετμημένῳ.</l>
+                <pb n="578" />
+                <l n="1002">Κροτωνιᾶται δʼ ἄστυ πέρσουσίν ποτε</l>
+                <l n="1003">Ἀμαζόνος, φθέρσαντες ἄτρομον κόρην</l>
+                <l n="1004">Κλήτην, ἄνασσαν τῆς ἐπωνύμου πάτρας.</l>
+                <l n="1005">πολλοὶ δὲ πρόσθεν γαῖαν ἐκ κείνης ὀδὰξ </l>
+                <l n="1006">δάψουσι πρηνιχθέντες, οὐδʼ ἄτερ πόνων</l>
+                <l n="1007">πύργους διαρραίσουσι Λαυρήτης γόνοι.</l>
+                <l n="1008">οἱ δʼ αὖ Τέρειναν, ἔνθα μυδαίνει ποτοῖς</l>
+                <l n="1009">Ὠκίναρος γῆν, φοῖβον ἐκβράσσων ὕδωρ,</l>
+                <l n="1010">ἄλῃ κατοικήσουσι κάμνοντες πικρᾷ. </l>
+                <l n="1011">τὸν δʼ αὖ τὰ δευτερεῖα καλλιστευμάτων</l>
+                <l n="1012">λαβόντα, καὶ τὸν ἐκ Λυκορμαίων ποτῶν</l>
+                <l n="1013">στρατηλάτην σῦν, καρτερὸν Γόργης τόκον,</l>
+                <l n="1014">τῇ μὲν Λίβυσσαν ψάμμον ἄξουσι πνοαὶ</l>
+                <l n="1015">Θρῇσσαι ποδωτοῖς ἐμφορούμεναι λίνοις, </l>
+                <l n="1016">τῇ δʼ ἐκ Λιβύσσης αὖθις ἐμπίπτων νότος</l>
+                <l n="1017">εἰς Ἀργυρίνους καὶ Κεραυνίων νάπας</l>
+                <l n="1018">ἄξει βαρεῖ πρηστῆρι ποιμαίνων ἅλα.</l>
+                <l n="1019">ἔνθα πλανήτην λυπρὸν ὄψονται βίον</l>
+                <l n="1020">Λακμωνίου πίνοντες Αἴαντος ῥοάς. </l>
+                <l n="1021">Κρᾶθις δὲ γείτων ἠδὲ Μυλάκων ὅροις</l>
+                <l n="1022">χῶρος <note n="1" place="unspecified">o)/ros (cod. B)
```

Comment:
another note. see previous for recommendations.

NB I have not looked at the print source and notes often have citations (titles in italic, bibl references, etc.) so there could be more encoding lurking in the note contents.

Beta code here needs a fix and tagging.